packages/date-picker/src/shared/context/SharedDatePickerContext.utils.ts

```diff
@@ -76,6 +101,21 @@ export const defaultSharedDatePickerContext: SharedDatePickerContextProps = {
   setInternalErrorMessage: () => {},
   clearInternalErrorMessage: () => {},
   autoComplete: AutoComplete.Off,
+  scrollContainer: {} as HTMLElement,
+  portalContainer: {} as HTMLElement,
+  portalClassName: '',
+  align: 'bottom',
+  justify: 'start',
+  spacing: 10,
+  adjustOnMutation: true,
+  popoverZIndex: 1,
+  onEnter: () => {},
+  onEntering: () => {},
+  onEntered: () => {},
+  onExit: () => {},
+  onExiting: () => {},
+  onExited: () => {},
+  contentClassName: '',
```

Comment:
Do we need to add these to the datepicker provider, or can we use Popover provider to access these values?

Reply:
Are you referring to this provider? I don't think we have defaults for `Popover` anywhere.

https://github.com/mongodb/leafygreen-ui/blob/e7d206e6c245b316b6ae8d2a419633612862deb3/packages/leafygreen-provider/src/PopoverContext/PopoverContext.tsx#L37-L53

Reply:
I made these props optional, so I no longer have to add them here.